src/explore-education-statistics-admin/src/pages/release/ReleaseDataGuidancePage.tsx

```diff
@@ -60,7 +60,7 @@ const ReleaseDataGuidancePage = ({
               renderDataCatalogueLink={
                 model.release.published ? (
                   <Link
-                    to={`${publicAppUrl}/data-catalogue?publicationId=${model.release.publicationId}&releaseId=${model.release.id}`}
+                    to={`${publicAppUrl}/data-catalogue?publicationId=${model.release.publicationId}&releaseVersionId=${model.release.id}`}
```

Comment:
This should still be `releaseId` as the url param hasn't been changed on the data catalogue page.